src/main/java/io/codearte/jfairy/producer/company/CompanyProvider.java

```diff
@@ -67,8 +68,27 @@ private void generateDomain() {
 		if (domain != null) {
 			return;
 		}
-		domain = TextUtils.stripAccents(StringUtils.strip(StringUtils.deleteWhitespace(name.toLowerCase()), ".").replace("/", ""))
-				+ "." + dataMaster.getRandomValue(DOMAIN);
+//		domain = TextUtils.stripAccents(StringUtils.strip(StringUtils.deleteWhitespace(name.toLowerCase()), ".").replace("/", ""))
```

Comment:
I've considered the CompanyProvider problem. I just have no idea that changing your default mechanism is proper or not. And if CompanyProvider is changed to Interface, maybe the other providers like PersonProvider should also be changed to keep a uniform design.

I can work on it in 2 or 3 days.

And I think it is better to keep generating a fixed url for each different company if possible. Maybe some people generate pair of company and url relying on this mechanism, but it may not be a big deal.